importer/Uploader.py

```diff
@@ -11,86 +11,34 @@ class Uploader(object):
-        """
-        Add labels and aliases.
-
-        Normally, if the WD item already has a label in $lang
-        and the data object has another one, the new one
-        will be automatically added as an alias. Otherwise
-        (no existing label), it will be added as a label.
-        However, the data model supports an optional [aliases]
-        field as well.
-        Its contents will be added directly as an alias, without checking
-        if it could be a label first.
-        """
-        print(labels)
-        for label in labels:
-            target_item.get()
-            label_content = label['value']
-            language = label['language']
-            self.wdstuff.addLabelOrAlias(
-                language, label_content, target_item)
-            if log:
-                t_id = target_item.getID()
-                message = "{} ADDED LABEL {} {}".format(
-                    t_id, language, label_content)
-                log.logit(message)
-        for alias in aliases:
-            target_item.get()
-            name = alias["value"]
-            language = alias["language"]
-            t_aliases = target_item.aliases
-            if language in t_aliases and name in t_aliases[language]:
-                return
-            else:
-                summary = "Added [{}] alias to [[{}]], {}".format(
-                    language, target_item.title(), self.summary)
-                target_item.editAliases(
-                    {language: [name]}, summary=summary)
-            if log:
-                t_id = target_item.getID()
-                message = "{} ADDED ALIAS {} {}".format(
-                    t_id, language, name)
-                log.logit(message)
+    def add_labels(self, target_item, labels, log):
+        """Add labels and aliases."""
+        labels_for_upload = {}
+        for language_key in labels:
+            label_content = labels[language_key]
+            labels_for_upload[language_key] = label_content
+        self.wdstuff.add_multiple_label_or_alias(
+            labels_for_upload, target_item)
+        if log:
+            t_id = target_item.getID()
+            message = "{} ADDED LABELS {}".format(
+                t_id, labels_for_upload)
+            log.logit(message)
 
     def add_descriptions(self, target_item, descriptions, log):
-        for description in descriptions:
-            target_item.get()
-            desc_content = descriptions[description]['value']
-            lang = descriptions[description]['language']
-            self.wdstuff.add_description(
-                lang, desc_content, target_item)
-            if log:
-                t_id = target_item.getID()
-                message = "{} ADDED DESCRIPTION {} {}".format(
-                    t_id, lang, desc_content)
-                log.logit(message)
+        """Add descriptions."""
+        descriptions_for_upload = {}
+        for language_key in descriptions:
```

Comment:
Actually how is `descriptions` different from `descriptions_for_upload` at the end of the loop?